setup.py

```diff
@@ -142,7 +152,7 @@ def get_version():
     extras_require={
         'all': ['ipython', 'matplotlib'],
     },
-    tests_require=['pytest', 'pytest-cov', 'numpy', 'scipy'],
+    tests_require=['pytest', 'pytest-cov', 'numpy', 'scipy', 'matplotlib', 'IPython<6'],
```

Comment:
Why pin to ipython < 6? Do we not want to support the latest stable ipython?
If this is really needed, suggest to at least leave a comment why this is done.